t/YachtSpecs.t

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::YachtSpecs)],
+    # At a minimum, be sure to include tests for all:
+    # - primary_example_queries
+    # - secondary_example_queries
+    'yacht maluhea' => test_spice(
+        '/js/spice/yacht_specs/query',
```

Comment:
This test fails as `/js/spice/yacht_specs/query` should be `/js/spice/yacht_specs/maluhea`